ui/src/main/js/page/dashboard/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+import React from 'react';
+
+//allow react dev tools work
+window.React = React;
+
+class Dashboard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  componentDidMount() {
+
+  }
+
+  render() {
+
+    return (
+        <div></div>
+    )
+  }
+};
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/modernmaster/forecaster/pullRequest?prid=7238741)
```suggestion
}
```